src/init.c

```diff
@@ -1209,32 +1209,26 @@ DLLEXPORT void julia_save()
     if (build_path) {
         if (jl_compileropts.compile_enabled == JL_COMPILEROPT_COMPILE_ALL)
             jl_compile_all();
-        char *build_ji;
-        if (asprintf(&build_ji, "%s.ji",build_path) > 0) {
-            jl_save_system_image(build_ji);
-            free(build_ji);
-            if (jl_compileropts.dumpbitcode == JL_COMPILEROPT_DUMPBITCODE_ON) {
-                char *build_bc;
-                if (asprintf(&build_bc, "%s.bc",build_path) > 0) {
-                    jl_dump_bitcode(build_bc);
-                    free(build_bc);
-                }
-                else {
-                    ios_printf(ios_stderr,"\nWARNING: failed to create string for .bc build path\n");
-                }
-            }
-            char *build_o;
-            if (asprintf(&build_o, "%s.o",build_path) > 0) {
-                jl_dump_objfile(build_o,0);
-                free(build_o);
+        ios_t *s = jl_create_system_image();
```

Comment:
I don't want to write both files by default; there should be an option to select which output format you want.